drivers/filesystems/cdfs_new/cdstruc.h

```diff
@@ -625,8 +672,42 @@ typedef struct _VCB {
 
     PVPB SwapVpb;
 
-} VCB;
-typedef VCB *PVCB;
+    //
+    //  Directory block cache. Read large numbers of blocks on directory
+    //  reads, hoping to benefit from the fact that most mastered/pressed
+    //  discs clump metadata in one place thus allowing us to crudely
+    //  pre-cache and reduce seeks back to directory data during app install, 
+    //  file copy etc.
+    //
+    //  Note that the purpose of this is to PRE cache unread data,
+    //  not cache already read data (since Cc already provides that), thus
+    //  speeding initial access to the volume.
+    //
+
+    PUCHAR SectorCacheBuffer;
+    CD_SECTOR_CACHE_CHUNK SecCacheChunks[ CD_SEC_CACHE_CHUNKS];
+    ULONG SecCacheLRUChunkIndex;
+    
+    PIRP SectorCacheIrp;
+    KEVENT SectorCacheEvent;
+    ERESOURCE SectorCacheResource;
+
+#ifdef CDFS_TELEMETRY_DATA
```

Comment:
This is Win8 related stuff. Make sure this isn't enabled at build.

Reply:
It's just #ifdefs, it's meant to be defined externally, so we should be good.